src/modules/feature/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { Applications, Feature } from './types'
+
+export const fetchFeaturesDelay = 10000
+
+export const fetchFeatures = async (applications: Applications[]) => {
```

Comment:
Do we want to work with the features of other applications at the same time?

Reply:
I think we do because maybe you are working in the marketplace and for some reason it might depend on a feature of the builder as well. For example, with stores v2 we'll be moving some functionality from the builder to the market and this might come handy as the feature might be in the "builder" namespace but also consumed in the market